ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx

```diff
@@ -123,173 +123,187 @@ export function VariableEditForm({
   }, [previewKey]);
 
   return (
-    <Box>
-      <SectionHeader>General</SectionHeader>
-      <Grid container spacing={2} mb={2}>
-        <Grid item xs={6}>
-          <TextField
-            required
-            error={!!validation.name}
-            fullWidth
-            label="Name"
-            value={state.name}
-            helperText={validation.name}
-            onChange={(v) => {
-              setState((draft) => {
-                draft.name = v.target.value as string;
-              });
+    <>
+      <Box
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          padding: (theme) => theme.spacing(1, 2),
+          borderBottom: (theme) => `1px solid ${theme.palette.grey[100]}`,
+        }}
+      >
+        <Typography variant="h2">Template Variables / Edit Variable</Typography>
+        <Stack direction="row" spacing={1} sx={{ marginLeft: 'auto' }}>
+          <Button
+            disabled={!validation.isValid}
+            variant="contained"
+            onClick={() => {
+              onChange(getVariableDefinitionFromState(state));
             }}
-          />
-        </Grid>
-        <Grid item xs={6}>
-          <FormControl fullWidth>
-            <InputLabel id="variable-type-select-label">Type</InputLabel>
-            <Select
-              labelId="variable-type-select-label"
-              id="variable-type-select"
-              label="Type"
-              value={state.kind}
+          >
+            Update
+          </Button>
+          <Button
+            variant="outlined"
+            onClick={() => {
+              onCancel();
+            }}
+          >
+            Cancel
+          </Button>
+        </Stack>
+      </Box>
```

Comment:
This `Box` is the header copied from `PanelDrawer`. I replaced the title and buttons.